iree_tests/onnx/import_tests.py

```diff
@@ -141,15 +142,25 @@ def import_onnx_files(test_dir_path, imported_dir_path):
             return False
         input_path = (imported_dir_path / test_input.stem).with_suffix(".npy")
         np.save(input_path, t)
-        test_data_flagfile_lines.append(f"--input=@{input_path.name}\n")
+        ss = getShapeString(t)
+        input_path_bin = (imported_dir_path / test_input.stem).with_suffix(".bin")
+        writeIOBinFile(t, input_path_bin)
+        test_data_flagfile_lines.append(f"--input={ss}=@{input_path_bin.name}\n")
     for i in range(len(test_outputs)):
         test_output = test_outputs[i]
         t = convert_io_proto(test_output, model.graph.output[i].type)
         if t is None:
             return False
         output_path = (imported_dir_path / test_output.stem).with_suffix(".npy")
         np.save(output_path, t)
```

Comment:
We can probably also delete the previous .npy files in this PR. I see both .npy and .bin files in https://github.com/nod-ai/SHARK-TestSuite/tree/iree_test_bin/iree_tests/onnx/node/generated/test_add_uint8, for example.

I think that's just a matter of dropping the `np.save` here and rerunning the importer, based on
```python

    # TODO(scotttodd): add flag to not clear output dir?
    print(f"Clearing old generated files from '{GENERATED_FILES_OUTPUT_ROOT}'")
    shutil.rmtree(GENERATED_FILES_OUTPUT_ROOT)
    GENERATED_FILES_OUTPUT_ROOT.mkdir(parents=True)
```

Reply:
Ah, I see this in the PR description

> keeps both npy and bin files for input and output for easy check of values using numpy

I guess that's okay. Could include a comment in the importer like this:
```python
np.save(output_path, t)  # Test using .bin but save .npy too for reference.
```